src/kakarot/instructions/system_operations.cairo

```diff
@@ -729,6 +731,7 @@ namespace CreateHelper {
         alloc_locals;
         let (state, account) = State.get_account(state, address);
         let nonce = account.nonce;
+        // Increment nonce of deployer account
```

Comment:
I don't really like this kind of doc really saying what can be read from the code

Reply:
I added this comment because it is surprising (but "efficient") that the `get_evm_address` method does mutate account of caller, 
I wanted to bring "visibility" to this surprising behaviour, though I will remove the comment.